notebooks/Roberts_Mock_data.ipynb

```diff
@@ -0,0 +1,1113 @@
+        "    \"Wilson\", \"Anderson\", \"Thomas\", \"Taylor\", \"Moore\", \"Jackson\", \"Martin\",\n",
+        "    \"Lee\", \"Perez\", \"Thompson\", \"White\", \"Harris\", \"Sanchez\", \"Clark\",\n",
+        "    \"Ramirez\", \"Lewis\", \"Robinson\", \"Walker\", \"Young\", \"Allen\", \"King\",\n",
+        "    \"Wright\", \"Scott\", \"Torres\", \"Nguyen\", \"Hill\", \"Flores\", \"Green\", \"Adams\",\n",
+        "    \"Nelson\", \"Baker\", \"Hall\", \"Rivera\", \"Campbell\", \"Mitchell\", \"Carter\",\n",
+        "    \"Roberts\", \"Gomez\", \"Phillips\", \"Evans\", \"Turner\", \"Diaz\", \"Parker\", \"Cruz\",\n",
+        "    \"Edwards\", \"Collins\", \"Reyes\", \"Stewart\", \"Morris\", \"Morales\", \"Murphy\",\n",
+        "    \"Cook\", \"Rogers\", \"Gutierrez\", \"Ortiz\", \"Morgan\", \"Cooper\", \"Peterson\",\n",
+        "    \"Bailey\", \"Reed\", \"Kelly\", \"Howard\", \"Ramos\", \"Kim\", \"Cox\", \"Ward\",\n",
+        "    \"Richardson\", \"Watson\", \"Brooks\", \"Chavez\", \"Wood\", \"James\", \"Bennet\",\n",
+        "    \"Gray\", \"Mendoza\", \"Ruiz\", \"Hughes\", \"Price\", \"Alvarez\", \"Castillo\",\n",
+        "    \"Sanders\", \"Patel\", \"Myers\", \"Long\", \"Ross\", \"Foster\", \"Jimenez\",\n",
+        ")\n",
+        "\n",
+        "state_names = (\n",
+        "    \"Alaska\", \"Alabama\", \"Arkansas\", \"American Samoa\", \"Arizona\", \"California\", \n",
+        "    \"Colorado\", \"Connecticut\", \"District of Columbia\", \"Delaware\", \"Florida\", \n",
+        "    \"Georgia\", \"Guam\", \"Hawaii\", \"Iowa\", \"Idaho\", \"Illinois\", \"Indiana\", \n",
+        "    \"Kansas\", \"Kentucky\", \"Louisiana\", \"Massachusetts\", \"Maryland\", \"Maine\",  \n",
+        "    \"Minnesota\", \"Missouri\", \"Mississippi\", \"Montana\", \"North Carolina\", \n",
+        "    \"North Dakota\", \"Nebraska\", \"New Hampshire\", \"New Jersey\", \"New Mexico\", \n",
+        "    \"Nevada\", \"New York\", \"Ohio\", \"Oklahoma\", \"Oregon\", \"Pennsylvania\", \n",
+        "    \"Puerto Rico\", \"Rhode Island\", \"South Carolina\", \"South Dakota\", \n",
+        "    \"Tennessee\", \"Texas\", \"Utah\", \"Virginia\", \"Virgin Islands\", \"Vermont\", \n",
+        "    \"Washington\", \"Wisconsin\", \"West Virginia\", \"Wyoming\", \"Michigan\",\n",
+        ")\n",
+        "\n",
+        "convictions = (\n",
+        "    \"Misdemeanors\", \"Felony\", \"Wobbler\",\n",
+        ")\n",
+        "\n",
+        "skills = (\n",
+        "    \"HTML\", \"CSS\", \"JavaScript\", \"Python\", \"Ruby\",\n",
+        "    \"C++\", \"Axios\", \"React\", \"Node.JS\", \"C#\", \"Django\",\n",
+        "    \"MongoDB\", \"SQL\", \"PostgreSQL\", \"NoSQL\", \"PHP\",\n",
+        "    \"Angular\", \"Microsoft Azure\", \"IOS\", \"Android\"\n",
+        ")\n",
+        "\n",
+        "time_zones = (\n",
+        "    \"Pacific\", \"Mountain\", \"Central\", \"Eastern\"\n",
+        "\n",
+        ")\n",
+        "\n",
+        "section_of_day = (\n",
```

Comment:
A little confused as to whether this, "section_of_day" is needed. Seems like it was somewhat overwritten by 'Time_of_day.'

Reply:
Yeah, I needed to remove this. Thanks for the catch! 😄